src/comps/main/addComments/addComments.jsx

```diff
@@ -16,41 +19,72 @@ export const AddComments = props => {
 
   function handleSubmit(e) {
     e.preventDefault();
-    // console.log(comment.hasOwnProperty("comment"));
-    if (false) {
-      console.log("cant send, the comment is not valid");
+
+    /** Name: isEmpty()
+     * function checks if an object is empty
+     * @param {obj} obj
+     * return boolean
+     */
+
+    const isEmpty = obj => {
+      for (let key in obj) {
+        if (obj.hasOwnProperty(key)) return false;
+      }
+      return true;
+    };
+
+    if (isEmpty(comment)) {
+      alert("The comment is not valid!");
     } else {
       addCommentFunc(comment)
-        .then(res => alert("added new comment..."))
+        .then(res => {
+          alert("added new comment!");
+          loadComments();
+        })
         .catch(err => console.log(err));
     }
   }
 
+  function loadComments() {
+    allComments()
+      .then(res => setCommentList(res.data))
+      .catch(err => console.log(err));
+  }
+
+  useEffect(() => {
+    loadComments();
+  }, []);
+  const style = {
+    btn: { marginLeft: "4px" }
```

Comment:
```suggestion
    submitButton: { marginLeft: "4px" }
```
no need to shorten everything